src/Google.Api.Gax/CallTiming.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    public enum CallTimingType
+    {
+        /// <summary>
+        /// Call uses retry, specified with a <see cref="RetrySettings"/>.
+        /// </summary>
+        Retry,
+
+        /// <summary>
+        /// Call does not use retry; call expiration is specified with an <see cref="Google.Api.Gax.Expiration"/> .
+        /// </summary>
+        Expiration,
+    }
+
+    public sealed class CallTiming
```

Comment:
Docs added.
Will leave it named CallTiming for now. Can reconsider the name later as required.